manifests/a/angeldev0/messyorganizer/1.0.0/angeldev0.messyorganizer.locale.en-US.yaml

```diff
@@ -0,0 +1,32 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: angeldev0.messyorganizer
+PackageVersion: 1.0.0
+PackageLocale: en-US
+Publisher: Angel C
+PublisherUrl: https://github.com/4ngel2769
+PublisherSupportUrl: https://github.com/4ngel2769
+Author: Angel C
+PackageName: MessyFileOrganizer
+PackageUrl: https://github.com/4ngel2769/messy-file-organizer
+License: MIT License
+Copyright: Angel C - 4ngel2769
+CopyrightUrl: https://angellabs.xyz
+ShortDescription: A simple downloads folder cleaner.
+Description: An application that sorts newly downloaded files into categories to fix your messy folders.
+Moniker: messyorganizer
+Tags:
+- messy
+- organuzer
+- organizer
+- file organizer
+- messyorganizer
+InstallationNotes: 'Thanks for installing MessyOrganizer! If you find any bugs in our app, please report them here: https://github.com/4ngel2769/messy-file-organizer/issues/new'
+Icons:
+- IconUrl: https://raw.githubusercontent.com/4ngel2769/messy-file-organizer/main/mfo.png
+  IconFileType: png
+  IconResolution: 256x256
+  IconTheme: default

```

Comment:
```suggestion
```

Reply:
As a note - I don't believe the message from the pipelines is accurate; It isn't that it requires VerifiedPublishers, but rather that it is not allowed by policy in this repo. Its just that the framework for VerifiedPublishers (which hasn't been finished yet) has been used to implement the policy